app-frontend/src/app/pages/library/projects/detail/projectScenes/projectScenes.html

```diff
@@ -14,57 +14,54 @@ <h1 class="h3 page-title">
                 ng-show="!$ctrl.$parent.selectedScenes.size && !$ctrl.isEditingProjectName && !$ctrl.isSavingProjectNameEdit">
           <i class="icon-pencil"></i>
         </button>
-        <div class="color-danger" ng-show="$ctrl.showError">
+        <div class="color-danger" ng-if="$ctrl.showError">
           There was an error renaming the project.
           <button class="btn btn-tiny" type="submit"
                   ng-click="$ctrl.dismissErrorMessage()">
             <i class="icon-cross color-danger"></i>
           </button>
         </div>
-        <div ng-show="$ctrl.isEditingProjectName">
+        <div ng-if="$ctrl.isEditingProjectName">
```

Comment:
the main thing is that ng-if used adding/ removing dom, while ng-show uses the ng-hide css - it's equivalent to `ng-class={'ng-hide': somefunc)}`, which allows you to do things like animations (which can also incidentally hide fuglies like flickering and is slightly more performant.